cli/engine/providers/aws/InfrastructureBuilder.py

```diff
@@ -220,26 +212,25 @@ def get_public_key(self):
         return public_key_config
 
     def add_security_rules_inbound_efs(self, infrastructure, security_group):
-        ags_allowed_to_efs = select_all(infrastructure, lambda item: item.kind == 'infrastructure/virtual-machine' and
+        vm_allowed_to_efs = select_all(infrastructure, lambda item: item.kind == 'infrastructure/virtual-machine' and
                                                       item.specification.authorized_to_efs)
 
-        for asg in ags_allowed_to_efs:
-            for subnet_in_asg in asg.specification.subnet_names:
-                subnet = select_single(infrastructure, lambda item: item.kind == 'infrastructure/subnet' and
-                                                                   item.specification.name == subnet_in_asg)
-
-                rule_defined = select_first(security_group.specification.rules, lambda item: item.source_address_prefix == subnet.specification.cidr_block
-                                                                                            and item.destination_port_range == 2049)
-                if rule_defined is None:
-                    rule = self.get_config_or_default(self.docs, 'infrastructure/security-group-rule')
-                    rule.specification.name = 'sg-rule-nfs-default-from-'+subnet.specification.name
-                    rule.specification.description = 'NFS inbound for '+subnet.specification.name
-                    rule.specification.direction = 'ingress'
-                    rule.specification.protocol = 'tcp'
-                    rule.specification.destination_port_range = "2049"
-                    rule.specification.source_address_prefix = subnet.specification.cidr_block
-                    rule.specification.destination_address_prefix = '*'
-                    security_group.specification.rules.append(rule.specification)
+        for vm in vm_allowed_to_efs:
```

Comment:
According to previous comment:
```suggestion
        for vm in vms_allowed_to_efs:
```

Reply:
Fixed